datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java

```diff
@@ -300,20 +304,28 @@ private CorpUserSnapshot extractUser(CorpuserUrn urn, CommonProfile profile) {
     return corpUserSnapshot;
   }
 
+  public static List<String> getGroupsClaimNames (OidcConfigs configs) {
+    List<String> groupsClaimNames = new ArrayList<>();
+    String rawGroupClaimsNames = configs.getGroupsClaimName();
+    if (rawGroupClaimsNames.startsWith("[") && rawGroupClaimsNames.endsWith("]")){
+      Type listType = new TypeToken<Collection<String>>(){}.getType();
+      groupsClaimNames = new Gson().fromJson(rawGroupClaimsNames, listType);
```

Comment:
We may want to consider handling the error that will result if this deserialization doesn't work into JSON and fail gracefully (with a warning logged and no groups)

Reply:
@jjoyce0510  I wasn't sure if you meant to do this instead of falling back to parsing comma separated groups? Aka. would we rather have it more strict in expecting JSON and if it fails, log that and just return no groups instead of "assuming" comma separated? Is that what you mean?